### PR TITLE
Pia 3182 add a loading indicator for the last added page on the analysis screen

### PR DESCRIPTION
**Pull request by vizaknaidavid**
simulator

Add custom loading indicator. 
Uncomment lines 92-92 in GiniBankSDKExample - Appcoordinator to test the custom loading

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift [hunk 1]
```diff
@@ -170,6 +170,14 @@ public final class ReviewViewController: UIViewController {
         return addPagesButton
     }()
 
+    private var loadingIndicatorView: UIActivityIndicatorView = {
+        let indicatorView = UIActivityIndicatorView()
+        indicatorView.hidesWhenStopped = true
+        indicatorView.style = .whiteLarge
```
Comment (zladzeyka) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift:176]: [Clarification] Will indicatorView.style = .whiteLarge work for iOS 12?

**Review by zladzeyka — COMMENTED** [review]
Great job!👏🏻🎉👍🏻

I've found an edge case when the loading indicator doesn't stop the animation 🥹. Switch off the multipage.

https://user-images.githubusercontent.com/3911816/196977015-6e5c02af-23ca-45a9-840a-547d777356c9.MP4

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift (hunk 1 above)
Comment (vizaknaidavid) [reply CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift:176]: It's the only one that works. `large` and `medium` were introduced in iOS 13.

**Review by zladzeyka — APPROVED** [review]
Looks good 👏🏻😊👍🏻

**Review by zladzeyka — APPROVED** [review]
👍🏻😉

**Pull request merged**